src/core/openthread.cpp

```diff
@@ -876,6 +878,33 @@ ThreadError otRemoveUnicastAddress(otInstance *, const otIp6Address *address)
     return sThreadNetif->RemoveExternalUnicastAddress(*static_cast<const Ip6::Address *>(address));
 }
 
+void otSlaacUpdate(otInstance *aInstance, otNetifAddress *aAddresses, uint32_t aNumAddresses,
+                   otSlaacIidCreate aIidCreate, void *aContext)
+{
+    Utils::Slaac::UpdateAddresses(aInstance, aAddresses, aNumAddresses, aIidCreate, aContext);
+}
+
+ThreadError otCreateRandomIid(otInstance *aInstance, otNetifAddress *aAddress, void *aContext)
+{
+    return Utils::Slaac::CreateRandomIid(aInstance, aAddress, aContext);
+}
+
+ThreadError otCreateMacIid(otInstance *aInstance, otNetifAddress *aAddress, void *aContext)
+{
+    (void) aInstance;
+    (void) aContext;
```

Comment:
No, I'm not saying remove the entire variable, just the names since you aren't using them. For instance:

`ThreadError otCreateMacIid(otInstance *, otNetifAddress *aAddress, void *)`

Then you don't need the `(void)aInstance` and `(void)aContext'

This is how most of openthread.cpp is using the `otInstance*` input variable right now.